util/io.go

```diff
@@ -17,13 +17,37 @@
 package util
 
 import (
+	"errors"
 	"fmt"
 	"io"
 	"os"
 	"path/filepath"
 	"strings"
+	"syscall"
 )
 
+func MoveFile(src, dst string) error {
+	err := os.Rename(src, dst)
+	var errno syscall.Errno
+	if err != nil && errors.As(err, &errno) && errno == syscall.EXDEV {
```

Comment:
I think it may be better to test whether it is an `os.LinkError` which should handle Windows as well.  I know we don't support it for real, but we probably don't want to break gratuitously.

The trouble is that there might be more reasons we get `LinkError`.  But on the third hand, who cares? It seems ok to try the manual path a bit more often than we "should".

Reply:
True, it doesn't cost much to try the copy and delete style if `os.Rename` fails. Implemented in d21751413